File: C_Programming/Study/Q5_Assignment13.c

///////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// Function Name :  Display
// Output:          Display elements which are multiple of 11.
// Description:     Assignment_13: Q.5  : Accept N Number from user & Display all such elements which are 
//                                        multiples of 11.
// Date:            05/06/2026
// Author:          Shital Ajit Nikam   
//
///////////////////////////////////////////////////////////////////////////////////////////////////////////

#include <stdio.h>
#include <stdlib.h>


void Display(int Arr[], int iLength)
{
    int iCnt = 0;

    for(iCnt = 0; iCnt < iLength; iCnt++)
    {
        if(Arr[iCnt]% 11 == 0)
        {
            printf("%d\t", Arr[iCnt]);
        }                       
    }
    
}

int main()
{
    int *p = NULL;
    int iSize = 0 , iCnt = 0;
   

    printf("Enter the number of elements \n");
    scanf("%d",&iSize);
    
    p = (int *)malloc(iSize * sizeof(int));

    if(p == NULL)
    {
        printf("Unable to allocate Memory");
        return -1;
    }

    printf("Enter %d Elements... \n",iSize);

    for(iCnt = 0; iCnt < iSize ; iCnt++)
    {
        printf("Enter Element: %d\n",iCnt+1);
        scanf("%d",&p[iCnt]);
    }

    printf("Elements which are multiples of 11 are: \n");
    Display(p, iSize);

    
    free(p);

    return 0;

}

/////////////////////////////////////////////////////////////////////////////////////
//Time Complexity : O(N)  where N = Number of elements
/////////////////////////////////////////////////////////////////////////////////////
// Input : N : 6
//        Elements : 85 66  3  55  93  88
// Output : 66  55  88
// 
// 
/////////////////////////////////////////////////////////////////////////////////////

File: Python_Programming/Competative/Q5_Assignment13.py
#///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
# Function Name :  GradeDisplay
#        
# Description:     Assignment_13: Q.4: Write a program which accepts Marks & Display grade. 
#                                      >=75 : Distinction, >=60 : First class, >=50 : Second Class, <50 : Fail                                  
#                                     
# Date:            30/05/2026
# Author:          Shital Ajit Nikam   
#
#///////////////////////////////////////////////////////////////////////////////////////////////////////////////////

def GradeDisplay(Marks):
    
    if(Marks >= 75):
        print("Grade is: Distinction") 
    elif(Marks >= 60):
        print("Grade is: First Class") 
    elif(Marks >= 50):
        print("Grade is: Second Class")
    else:
        print("Grade is: Fail !!")          
    
        
def main():
    value = int(input("Enter the Marks: "))    
        
    GradeDisplay(value)        
              
    
if __name__ == "__main__":
    main()